### PR TITLE
Lift off

### PR DESCRIPTION
**Pull request by jimmywarting**
Added some basic functionality so we get the get started shaping conflux

### CODE REVIEW

**Comment by jimmywarting** [comment]
Maybe this will be a good starting point or something?
maybe should put everything in a `src` dir

File: example/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="module">
```
Comment (jimmywarting) [inline example/index.html]: I would sometime like to try out ES module, Maybe we can support both worlds

We can maybe this kind of stuff to support both way:
```html
<script type="module" src="app.js"></script>
<script nomodule src="classic-app-bundle.js"></script>
```

using modules in browsers directly is also part of why i don't like TS

**Comment by eligrey** [comment]
lgtm 👍

File: example/index.html (hunk 1 above)
Comment (bencmbrook) [reply example/index.html]: Adding a CD deployed site of example/index.html so we can preview it in our PRs

Comment (bencmbrook) [reply example/index.html]: https://conflux.netlify.com/

**Pull request merged**